README.md

```diff
@@ -22,9 +22,10 @@ headers, and XML registry.
 
 ### Assembler, binary parser, and disassembler
 
-* Based on SPIR-V 1.0 Revision 3.
-  * Supports GLSL std450 extended instructions.
-  * Supports OpenCL extended instructions.
+* Based on SPIR-V 1.1, including extended instruction sets:
```

Comment:
The SPIR-V version is decoupled from the extended instruction sets.
Better to list the extended instruction sets separately.

Also, I'd say SPIR-V 1.1 Rev 1  since in the future we might apply fixes from future revs, and someone should be able to tell whether those fixes are applied by seeing that we've rolled to a newer Rev.